### PR TITLE
chore: Adds Appsmith to the project list

### PR DESCRIPTION
**Pull request by megaconfidence**
This PR adds Appsmith to the Up-for-grabs list. 
Please, let me know if you have any questions or feedback.

### CODE REVIEW

File: _data/projects/Appsmith.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+tags:
+  - low-code
+  - internal-tools
+  - crud-apps
+  - admin-panels
+  - app-builder
+  - admin-dashboard
+upforgrabs:
+  name: Good First Issue
+  link: https://github.com/appsmithorg/appsmith/issues?q=is%3Aopen+is%3Aissue+label%3A%22Good+First+Issue%22+bug
```
Comment (ritwik12) [inline _data/projects/Appsmith.yml]: @cokoghenun Please use this instead https://github.com/appsmithorg/appsmith/labels/Good%20First%20Issue

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/Appsmith.yml (hunk 1 above)
Comment (megaconfidence) [reply _data/projects/Appsmith.yml]: I've updated this @ritwik12

Comment (ritwik12) [reply _data/projects/Appsmith.yml]: Thank you

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@cokoghenun Thanks!!

**Pull request merged**